First review for translation of: Softwarereview_intro.pt.Rmd

beatrizmilz opened the pull request:
Review of the translation for this document: https://raw.githubusercontent.com/ropensci/dev_guide/refs/heads/main/softwarereview_intro.Rmd

beatrizmilz commented on softwarereview_intro.pt.Rmd:
```diff
@@ -1,31 +1,31 @@
-Tecnicamente, aproveitamos ao máximo o [GitHub](https://github.com/) cada processo de revisão de pacote é um problema no diretório [ropensci/software-review no repositório do GitHub](https://github.com/ropensci/software-review/). Por exemplo, clique em [aqui](https://github.com/ropensci/software-review/issues/24) para ler o tópico de avaliação do `ropenaq` pacote: o processo é uma conversa contín ...
+Tecnicamente, aproveitamos ao máximo o [GitHub](https://github.com/): cada processo de revisão de pacote é uma *issue* no [repositório `ropensci/software-review` no GitHub](https://github.com/ropensci/software-review/). Por exemplo, clique em [aqui](https://github.com/ropensci/software-review/issues/24) para ler o tópico de avaliação do pacote `ropenaq`: o processo é uma conversa contínua até a a ...
 
 ## Por que enviar seu pacote para a rOpenSci? {#whysubmit}
```
Revisei até aqui!

yabellini replied:
Ola @beatrizmilz is coming here to ask about the status of this review.  Do you think you will have the time to finished?  Thanks in advance!

beatrizmilz replied:
> Ola @beatrizmilz is coming here to ask about the status of this review. Do you think you will have the time to finished? Thanks in advance!

Hi Yani! I think I started it on the translaton and forgot to add on my to-do list!
I just added, I can finish it soon!

maelle commented on softwarereview_intro.pt.Rmd:
```diff
@@ -47,17 +47,17 @@ Além disso, os autores e revisores podem entrar em contato com os editores em p
 - O repositório do seu pacote será transferido para a organização rOpenSci.
```
```suggestion
- Se você quiser, o repositório de seu pacote será transferido para a organização rOpenSci.
```

@beatrizmilz this is the change made in #864, could we please add it for Portuguese?

beatrizmilz commented:
There are some things that I'm noticing but I don't want to change the text, so I'll leave it as a comment here so it is easier for us to detect and discuss. Ok?

beatrizmilz commented on softwarereview_intro.pt.Rmd:
```diff
@@ -1,31 +1,31 @@
+A [coleção](https://ropensci.org/packages/) da rOpenSci é parcialmente resultado de contribuições de  membros(as) da equipe e parcialmente constituída por membros(as) da comunidade, o que significa que a coleção provém de uma grande diversidade de habilidades e de experiência de pessoas desenvolvedoras. Como garantir a qualidade de todo a coleção? É aí que entra a revisão por pares de software: o ...
 
-Você pode reconhecer os pacotes do rOpenSci que foram revisados por pares por meio de um emblema verde "peer-reviewed" em seu README, com links para suas revisões (cf [este exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição em [página de pacotes do rOpenSci](https://ropensci.org/packages/) com links para as revis ...
+Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um ícone de comentário azul próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com link ...
```
It was quite difficult to understand what it meant in the translation.
I looked up on the website and this is an attempt to describe it a bit better.
But this is a change from the source book, I'm not sure how to proceed.

```suggestion
Você pode reconhecer os pacotes da rOpenSci que foram revisados por pares por meio de um selo (*badge*) verde com o texto "peer-reviewed" em seu README, com links para suas revisões ([por exemplo](https://github.com/ropensci/restez#locally-query-genbank-)); e por meio de um botão azul com o texto *"Peer-reviewed"* próximo à sua descrição na [página de pacotes da rOpenSci](https://ropensci.org/packages/) com links para as revisões.
```

beatrizmilz commented on softwarereview_intro.pt.Rmd:
```diff
@@ -44,20 +44,20 @@ Além disso, os autores e revisores podem entrar em contato com os editores em p
 - O LEIAME do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
 - Seu pacote receberá um selo [`docs.ropensci.org` site de documentos](#rodocsci) que você pode vincular a partir da DESCRIÇÃO.
-- O repositório do seu pacote será transferido para a organização rOpenSci.
+- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.
```
Making more clear what we mean by transfering to rOpenSci org. 
But this is not in the English version, check how to proceed.
```suggestion
- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci no GitHub.
```

beatrizmilz commented on softwarereview_intro.pt.Rmd:
````diff
@@ -73,6 +73,8 @@ source(file.path("scripts", "airtable-access.R"), local = knitr::knit_global())
 source(file.path("scripts", "airtable-get-reviewers.R"), local = knitr::knit_global())
 ```
 
+### Editores(as) Convidados(as) {#guesteditors}
````
This is not in the original version (english). But to follow the structure, I added this h3-title. 

Let me know what do you think, Yani. We can also edit it in English and Spanish.

beatrizmilz commented on softwarereview_intro.pt.Rmd:
```diff
@@ -1,63 +1,63 @@
+- O README do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
+- Seu pacote terá um [site de documentação em `docs.ropensci.org`](#rodocsci) que você pode referenciar no DESCRIPTION.  
+- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.
+- Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), seus metadados mencionarão a revisão.
```
The phrase is not 100% clear to me.
In "their metadata will mention the review.", "their metadata" is referring to the package metadata? 
If so, we can make if more explicit:
```suggestion
- Se os(as) revisores(as) [concordarem em ser listados(as) no DESCRIPTION](#authorship), os metadados do pacote mencionarão a revisão.
```

yabellini commented:
@beatrizmilz I updated the other branch, but it looks like this does not solve the problem (and even adds some conflicts).

To have a cleaner PR, can you create a new one that only has the changes to the file softwarereview_intro.pt.Rmd ?

Thank you!

beatrizmilz commented:
> @beatrizmilz I updated the other branch, but it looks like this does not solve the problem (and even adds some conflicts).
> 
> To have a cleaner PR, can you create a new one that only has the changes to the file softwarereview_intro.pt.Rmd ?
> 
> Thank you!

Hi Yani!
Thank you for the response. I have moved all the comments and suggestions to this existing PR: #822 

Let me know if this works!

yabellini commented:
@beatrizmilz Thank you! Yes it works!  I will close this one now.
